2020-05-11 20:35:43,766 - instabot version: 0.117.0 (bot) - INFO - Instabot version: 0.117.0 Started
2020-05-11 20:35:43,766 - instabot version: 0.117.0 (bot) - DEBUG - Bot imported from C:\Users\Dinopc\PycharmProjects\COVIDBot\venv\lib\site-packages\instabot\bot\bot.py
2020-05-11 20:35:43,767 - instabot version: 0.117.0 (api_login) - DEBUG - Loading cookies
2020-05-11 20:35:43,767 - instabot version: 0.117.0 (api_login) - DEBUG - Loading uuids
2020-05-11 20:35:43,767 - instabot version: 0.117.0 (api_login) - INFO - Recovery from C:\Users\Dinopc\PycharmProjects\COVIDBot/config/covidnews_br_uuid_and_cookie.json: COOKIE True - UUIDs True - TIMING, DEVICE and ...
- user-agent=Instagram 117.0.0.28.123 Android (28/9.0; 420dpi; 1080x1920; OnePlus; ONEPLUS A3003; OnePlus3; qcom; en_US; 180322800)
- phone_id=34d2b071-c184-485a-9c30-d3edf26c987a
- uuid=5b692fe7-879c-409e-a261-3b2ff54ec789
- client_session_id=58b80178-9751-49b0-babe-4ac2e29b1542
- device_id=android-3c58beea084e55de
2020-05-11 20:35:43,777 - instabot version: 0.117.0 (api_login) - INFO - LOGIN FLOW! Just logged-in: False
2020-05-11 20:35:44,494 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/timeline/ returned response: <Response [200]>
2020-05-11 20:35:44,873 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: feed/reels_tray/ returned response: <Response [200]>
2020-05-11 20:35:44,879 - instabot version: 0.117.0 (api) - INFO - Logged-in successfully as 'covidnews_br'!
2020-05-11 20:35:44,879 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Loading Checkpoint file from: C:\Users\Dinopc\PycharmProjects\COVIDBot/config/covidnews_br.checkpoint
2020-05-11 20:36:02,715 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: media/configure/? returned response: <Response [400]>
2020-05-11 20:36:02,715 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "Uploaded image isn\'t in the right format", "status": "fail"}'
2020-05-11 20:36:02,715 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2020-05-11 20:36:02,715 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: Uploaded image isn't in the right format
2020-05-11 20:36:02,715 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2020-05-11 20:36:18,074 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: media/configure/? returned response: <Response [400]>
2020-05-11 20:36:18,074 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "Uploaded image isn\'t in the right format", "status": "fail"}'
2020-05-11 20:36:18,074 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2020-05-11 20:36:18,074 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: Uploaded image isn't in the right format
2020-05-11 20:36:18,074 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2020-05-11 20:36:33,442 - instabot version: 0.117.0 (api) - DEBUG - POST to endpoint: media/configure/? returned response: <Response [400]>
2020-05-11 20:36:33,442 - instabot version: 0.117.0 (api) - DEBUG - Responsecode indicates error; response content: b'{"message": "Uploaded image isn\'t in the right format", "status": "fail"}'
2020-05-11 20:36:33,442 - instabot version: 0.117.0 (api) - ERROR - Request returns 400 error!
2020-05-11 20:36:33,442 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: Uploaded image isn't in the right format
2020-05-11 20:36:33,442 - instabot version: 0.117.0 (api) - INFO - Instagram's error message: {}
2020-05-11 20:36:48,446 - instabot version: 0.117.0 (bot_checkpoint) - DEBUG - Saving Checkpoint file to: C:\Users\Dinopc\PycharmProjects\COVIDBot/config/covidnews_br.checkpoint
2020-05-11 20:36:48,451 - instabot version: 0.117.0 (bot) - INFO - Total requests: 5
